ophyd/mca.py

```diff
@@ -0,0 +1,153 @@
+from __future__ import print_function
+import logging
+
+from collections import OrderedDict
+
+from .signal import (EpicsSignal, EpicsSignalRO)
+from .device import Device
+from .device import Component as C, DynamicDeviceComponent as DDC
+from .areadetector import EpicsSignalWithRBV as SignalWithRBV
+
+
+logger = logging.getLogger(__name__)
+
+
+class ROI(Device):
+
+    # 'name' is not an allowed attribute
+    label = C(EpicsSignal, 'NM', lazy=True)
```

Comment:
Thinking aloud: maybe we can test to see if someone tries overriding `OphydObject` `name`/etc with components.